docker/alpine/Dockerfile

```diff
@@ -9,6 +8,7 @@
 # (see CMakeLists.txt for current version number) -
 #
 #   docker buildx build \
+#     -f docker/ubuntu/Dockerfile \
 #     --platform linux/amd64,linux/arm64 \
 #     --tag mtconnect/agent:2.0.0.12_RC18 \
```

Comment:
Either remove `2.0.0.12_RC18` or replace it with `latest` or update it on each release.

```suggestion
#     --tag mtconnect/agent:latest \
```